roles/certs/generate-csr/README.md

```diff
@@ -0,0 +1,71 @@
+generate-csr
+============
+
+This ansible role is used to create a CSR (Certificate Signing Request) for use with CA to generate valid certificates.
+
+Requirements
+------------
+
+- An operational IdM/IPA account with the proper permissions to manage hosts.
+
+Role Variables
+--------------
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|csr_host_name| The hostname/FQDN of the host | yes | |
+|csr_country| CSR Country (e.g.: 'US') | yes | |
+|csr_state| CSR State (e.g.: North Carolina) | yes | |
+|csr_location| CSR Location/City/Town (e.g.: Raleigh) | yes | |
+|csr_org_name| CSR Organization/Company Name (e.g.: Red Hat, Inc.) | yes | |
+|csr_org_unit| CSR Organization Name (e.g.: Community of Practice) | yes | |
+|csr_email| CSR E-mail address to be used | true | |
+|csr_subject_alt_names| List of CSR SANs | false | |
+
+
+
+Example Playbooks
+----------------
+
+```
+- hosts: csr-host
+  roles:
+  - generate-csr
+```
+
+Example Inventory
+----------------
+
+**inventory/hosts**
+```
+[csr-host]
```

Comment:
Probably should be cert-host to match what's in the other playbooks/roles

Reply:
no, I'd like to keep it different to "showcase" another way - i.e.: don't make it look like it **has** to be the same